com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/bundle/test/BundlerTest.java

```diff
@@ -200,6 +200,13 @@ else if (platform == Platform.WasmWeb)
             File wasmFile = new File(outputDirFile, exeName + ".wasm");
             checkFileExist(outputDirFile, wasmFile);
         }
+        else if (platform == Platform.WasmPthreadWeb)
+        {
+            File wasmjsFile = new File(outputDirFile, exeName + "_pthread_wasm.js");
+            checkFileExist(outputDirFile, wasmjsFile);
+            File wasmFile = new File(outputDirFile, exeName + "_pthread.wasm");
+            checkFileExist(outputDirFile, wasmFile);
+        }
```

Comment:
pthread->pthreads. Probably it's worth be fixed in the main branch during testing.

Reply:
I agree it's a bit confusing, but I chose `pthread` after the specific compiler flag.
And as the platform is called "pthread", I chose "pthread" for the output files.

We still have time to rename the platform/feature etc.